### PR TITLE
Add extra metadata to __source prop

### PR DESCRIPTION
**Pull request by zikaari**
__source prop will now have column number along with line number of opening element
as well as line number of closing element

| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | NA
| Patch: Bug Fix?          | No
| Major: Breaking Change?  | Might be
| Minor: New Feature?      | Yes
| Tests Added + Pass?      | Yes
| Documentation PR         | No
| Any Dependency Changes?  | No
| License                  | MIT

I'm working on an extension for chrome devtools, that's also devtools in itself.

Basically it's devtools' workspaces done right.

Along with giving devs a text editor right inside devtools, that automagically points to project directory, they can literally pick any element on the page and it'll take them to the `file:line:col` where that element originated.

While I work on Vue and Angular support, this PR will probably be the last nail in wrapping up React support

### CODE REVIEW

**Comment by NeekSandhu** [comment]
@hzoo Any hurdles in moving forward with this?

**Comment by nicolo-ribaudo** [comment]
cc @babel/react Does React support those additional metadata?
